### ISSUE TITLE
wxListView  fail to refresh items when DeleteAllItems is called

### ISSUE THREAD

**Issue by Migow50** [wxWidgets/wxWidgets#24956]
on windows 10 22H2  withwxwidget 3.2.1 wxListView(wxLC_REPORT | wxLC_EDIT_LABELS)   fail(we can't select the item but it is still drawn) to refresh items when DeleteAllItems is called but if DeleteItem is called refresh is correct, calling RefreshItems does not work as a fix 

as a workaround fix i create an item(InsertItem) then delete it (DeleteItem), it force refresh the list

 i will provide a test code if necessary

**Comment by vadz** [comment]
Yes, please [make a patch](https://www.wxwidgets.org/develop/how-to-submit-patches/) to the listctrl sample reproducing the problem, if only to allow us to check if it's still present in the latest version.

Thanks!
